src/types/EmbeddedPaymentElement.tsx

```diff
@@ -100,6 +100,12 @@ export interface EmbeddedPaymentElementConfiguration {
   merchantDisplayName: string;
   /** The identifier of the Stripe Customer object. See https://stripe.com/docs/api/customers/object#customer_object-id */
   customerId?: string;
+  /** A short-lived token that allows the SDK to access a Customer’s payment methods. */
+  customerEphemeralKeySecret?: string;
+  /** (Experimental) This parameter can be changed or removed at any time (use at your own risk).
```

Comment:
I think this is GA'd now!

Reply:
I see CustomerSessions GA still in-flight [here](https://docs.google.com/spreadsheets/d/1KgnHg_wIhTILROsOuorDWo8pcnIqFS10YZ59mIm9pXs/edit?pli=1&gid=1161657454#gid=1161657454), but asking in Slack.

Reply:
Looks like no GA yet. https://stripe.slack.com/archives/CFA2HJ99A/p1747756997653869?thread_ts=1747756658.290449&cid=CFA2HJ99A